quarkus.ls/com.redhat.quarkus.ls/src/main/java/com/redhat/quarkus/services/QuarkusValidator.java

```diff
@@ -130,13 +130,61 @@ private void validatePropertyValue(String propertyName, ExtendedConfigDescriptio
 			return;
 		}
 
-		// TODO validate boolean, int, enums types
-		String type = metadata.getType();
+		String value = property.getValue().getText().trim();

```

Comment:
Replace with 

```java
String value = property.getPropertyValue();
if (value == null || value.isEmpty()) {
	// TODO: validate required.
	return;
}
```

and trim (if required should be done in property.getPropertyValue())